app/controllers/static_pages_controller.rb

```diff
@@ -1,2 +1,17 @@
 class StaticPagesController < ApplicationController
+
+  def about
+    @teamMembers = Member.where(role: 1)
+    @advisoryMembers = Member.where(role: 2)
+
+    gon.team = serialized(@teamMembers).to_json
```

Comment:
Ok, so your parse the `gon` properties in the front because you "jsonify" it in the server. You don't have to!

See some examples in CMS: https://github.com/Vizzuality/forest-atlas-landscape-cms/blob/develop/app/controllers/site_page_controller.rb#L83-L103